js/test/BTCUtils.test.js

```diff
@@ -1,442 +1,317 @@
 /* global it describe BigInt */
 import * as chai from 'chai';
-import * as utils from '../utils/utils';
+import * as utils from '../src/utils';
 import * as BTCUtils from '../src/BTCUtils';
 import * as vectors from '../../testVectors.json';
 
-let vectorObj = JSON.parse(JSON.stringify(vectors));
-
-utils.parseJson(vectorObj)
-
-let {
-  HEADER_170,
-  OP_RETURN_PROOF,
-  OP_RETURN_INDEX,
-  OUTPOINT,
-  HASH_160,
-  HASH_256,
-  SEQUENCE_WITNESS,
-  SEQUENCE_LEGACY,
-  OUTPUT,
-  INDEXED_INPUT,
-  INDEXED_OUTPUT,
-  LEGACY_INPUT,
-  INPUT_LENGTH,
-  SCRIPT_SIGS,
-  SCRIPT_SIG_LEN,
-  INVALID_VIN_LEN,
-  INVALID_VOUT_LEN,
-  OUTPUT_LEN,
-  HEADER,
-  MERKLE_ROOT,
-  OP_RETURN,
-  TWO_IN,
-  RETARGET_TUPLES
+const vectorObj = JSON.parse(JSON.stringify(vectors));
```

Comment:
why do we need this step? :thinking:

Reply:
Need to convert to object, then iterate over object and return updated object.  Is there a better way to do this?